thirdparty/.gitignore

```diff
@@ -0,0 +1 @@
+angelscript
```

Comment:
Guess one script lang binding does the job for a game that doesn't even work. :D